doc/user_guide/io.rst

```diff
@@ -365,4 +367,43 @@ Therefore strongly reccommend to load using the ``.emi`` file instead.
 When reading an ``.emi`` file if there are several ``.ser`` files associated
 with it, all of them will be read and returned as a list.
 
+.. _blockfile-format:
+
+Blockfile
+---------
+
+HyperSpy can read and write the blockfile format from NanoMegas ASTAR software.
+It is used to store a series of diffraction patterns from scanning precession
+electron difraction (SPED) measurements, with a limited set of metadata. The
+header of the blockfile contains information about centering and distortions
+of the diffraction patterns, but is not applied to the signal during reading.
+
+.. warning::
```

Comment:
I agree on the datatype warning. Regarding limitations of guarantees, I'm not sure I see a scenario where this would be relevant. If somebody overwrites their data using this, and then blames us, I doubt having a notice in the docs will make a difference. If its just to limit liability, there's also this in the license:
> HyperSpy is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

:P
I would agree if this write-support was somehow experimental or incomplete, but at the moment there's basically only one header field that is partially unknown, and that we treat in a quite safe manner.